packages/@react-types/searchwithin/src/index.d.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {DOMProps, SpectrumLabelableProps, StyleProps} from '@react-types/shared';
+import {ReactNode} from 'react';
+
+export interface SpectrumSearchWithinProps extends SpectrumLabelableProps, DOMProps, StyleProps {
+  /** The SearchField and Picker contained within the SearchWithin. */
+  children: ReactNode,
```

Comment:
Maybe `[ReactElement, ReactElement]` to be more specific? Just following the precedent set by MenuTrigger/DialogTrigger

Reply:
Great catch